modules/cf-core/src/store.ts

```diff
@@ -215,7 +216,8 @@ export class Store {
     myIdentifier: string,
     theirIdentifier: string,
     proxyFactoryAddress: string,
-    minimumViableMultisigAddress: string
+    minimumViableMultisigAddress: string,
+    acceptGeneratedMultisig: boolean // = false,
```

Comment:
Why not default to false?

Reply:
that was temporary, added so the compiler would  point me to all uses. I wanted to go through them one by one in this case, but it now does default to false